sqls/activity-repo-top-Chinese/pre-processor.js

```diff
@@ -0,0 +1,20 @@
+module.exports = async function(config) {
+    let actor_ids = new Array();
+    for (let i in config.actors) {
+        actor_ids.push(config.actors[i].id);
+    }
+    let org_ids = new Array();
+    for (let i in config.orgs) {
+        org_ids.push(config.orgs[i].id);
+    }
+    let repo_ids = new Array();
+    for (let i in config.repos) {
+        repo_ids.push(config.repos[i].id);
+    }
+    let ids={
+        "actor_ids":actor_ids,
```

Comment:
You can not use array here, you need to construct a string.

Reply:
OK.

Reply:
Actually I think it's not about the string or array, it's because wrong param in SQL template